### PR TITLE
DP-2463 typos + redirects

### PR DESCRIPTION
**Pull request by Mara3l**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: docs/content/en/docs/workspace/workspaces/load_and_put_declarative_workspaces.md [hunk 1]
```diff
@@ -36,15 +36,13 @@ sdk.catalog_workspace.load_and_put_declarative_workspaces(
 
 Or by two separate calls:
 
-```Python
+```python
```
Comment (hkad98) [inline docs/content/en/docs/workspace/workspaces/load_and_put_declarative_workspaces.md:39]: Why did you rename this and not ```Python above? ;) Can you check the rest of files in docs as well?

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: docs/content/en/docs/workspace/workspaces/load_and_put_declarative_workspaces.md (hunk 1 above)
Comment (Mara3l) [reply docs/content/en/docs/workspace/workspaces/load_and_put_declarative_workspaces.md:39]: fixed

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Review by hkad98 — APPROVED** [review]

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-python-sdk-gate-PR274

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-fossa-scanning-zuul/97/ : FAILURE in 1m 16s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-sonarqube-scan-zuul-gate/247/ : SUCCESS in 52s
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-merge-compliance-check/250/ : SUCCESS in 13s
